app/controllers/application_controller.rb

```diff
@@ -2,8 +2,25 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+  around_filter :set_time_zone
 
   def authenticate_admin!
     redirect_to new_user_session_path unless current_user && current_user.admin?
   end
+
+  private
+
+    def set_time_zone
+      begin
```

Comment:
So the around filter wraps the entire request, unlike a [before or after] filter. Everything between `begin` and `yield` acts as a `before filter` and everything after that acts like an `after filter`.